sos/plugins/kubernetes.py

```diff
@@ -22,33 +23,101 @@ class kubernetes(Plugin, RedHatPlugin):
     """Kubernetes plugin
     """
 
-    option_list = [("podslog", "capture logs for pods", 'slow', False)]
+    # Red Hat Atomic Platform and OpenShift Enterprise use the
+    # atomic-openshift-master package to provide kubernetes
+    packages = ('kubernetes', 'atomic-openshift-master')
```

Comment:
In a containerized install there are no packages visible from the host, relevant packages are are inside the container's image (think e.g. an install on Atomic Host where `/usr` is read-only and there is no `yum`). Configuration files are kept on the host and bind-mounted into the container at runtime.